nservicebus/msmq/routing_content_core_[6,).partial.md

```diff
@@ -11,6 +11,8 @@ NOTE: When running under ASP.NET the `instance-mapping.xml` file may not be loca
 
 snippet: InstanceMappingFile
 
+NOTE: The endpoint names are case sensitive e.g. if endpoint name is used in routing API and the instance mapping file, the file entries will be used only when there is a case sensitive match. 
```

Comment:
I don't have strong feelings about this. Maybe rephrase to something like `The endpoint names are case sensitive. Endpoint names used in the routing name and the instance mapping file require an exact string match` to avoid the `e.g.` ?